deploy.sh

```diff
@@ -11,11 +11,13 @@ if [ "$1" = "prod" ]; then
   read answer
   if [ "$answer" != "${answer#[Yy]}" ] ;then
     scp -r dist/* $2@athena.dialup.mit.edu:/mit/courseroad/web_scripts/courseroad/
+    scp -r .htaccess $2@athena.dialup.mit.edu:/mit/courseroad/web_scripts/courseroad/
   else
     echo cancelled
   fi
 elif [ "$1" = "dev" ]; then
     scp -r dist/* $2@athena.dialup.mit.edu:/mit/courseroad/web_scripts/courseroad/dev/
+    scp -r .htaccess.dev $2@athena.dialup.mit.edu:/mit/courseroad/web_scripts/courseroad/dev/
```

Comment:
will the file `.htaccess.dev' be recognized on courseroad/dev/ ? I think it needs to be renamed?

Reply:
Oops, thanks for catching this.